items/shields.xml

```diff
@@ -1,2208 +1,1792 @@
 ﻿<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <Items>
-  <Item id="crpg_leather_round_shield_v2_h0" name="{=WYFDXE82}Large Round Shield" body_name="bo_cap_sturgia_shield_a" shield_body_name="bo_sturgia_shield_b" recalculate_body="false" mesh="sturgia_shield_b" using_tableau="true" weight="6.0306" difficulty="0" appearance="1" Type="Shield" item_holsters="shield_round:shield_4" has_lower_holster_priority="true" holster_position_shift="0,0,0.01" culture="Culture.sturgia">
+  <Item id="crpg_adarga_v3_h0" name="{=Diggles}Desert Adarga Round Shield" body_name="bo_cap_shield_aserai_cavalary_a" shield_body_name="bo_shield_aserai_cavalary_a" recalculate_body="false" mesh="shield_aserai_cavalary_a" using_tableau="true" weight="0.6903125" appearance="0.1" Type="Shield" item_holsters="shield:shield_2:shield_3:shield_4" has_lower_holster_priority="true" holster_position_shift="-0.1,-0.0,0" culture="Culture.aserai">
     <ItemComponent>
-      <Weapon weapon_class="LargeShield" body_armor="5" thrust_speed="89" thrust_damage="5" thrust_damage_type="Blunt" speed_rating="89" physics_material="wood_shield" item_usage="hand_shield" position="-0.02, 0.0, 0.0" rotation="-15.0,-90.0,-10.0" weapon_length="92" center_of_mass="0, 0.15, 0" hit_points="570" modifier_group="shield">
+      <Weapon weapon_class="SmallShield" body_armor="5" thrust_speed="82" thrust_damage_type="Blunt" speed_rating="82" physics_material="wood_shield" item_usage="shield" position="0.0, 0.00, 0.00" rotation="0.0,10.0,40.00" weapon_length="47" center_of_mass="-0.0,0.0,0.05" hit_points="250" modifier_group="shield">
         <WeaponFlags CanBlockRanged="true" HasHitPoints="true" />
       </Weapon>
     </ItemComponent>
-    <Flags WoodenParry="true" HeldInOffHand="true" ForceAttachOffHandPrimaryItemBone="true" />
+    <Flags WoodenParry="true" HeldInOffHand="true" ForceAttachOffHandSecondaryItemBone="true" />

```

Comment:
this is unreadable as you've changed the order of the items.
I cannot know what you changed because everything is shuffled
Do a pull request with only the order change , then rebase this one.